users/admin.py

```diff
@@ -20,16 +21,23 @@ class UserAdmin(VersionAdmin, DjangoUserAdmin):
         "is_staff",
     )
     search_fields = ("uuid", "first_name", "last_name", "email")
-    inlines = [ProfileAdminInline]
 
     def get_fieldsets(self, request, obj=None):
         fieldsets = super().get_fieldsets(request, obj)
         if obj:
             fieldsets = deepcopy(fieldsets)
-            fieldsets[1][1]["fields"] += ("uuid",)
+            fieldsets[1][1]["fields"] += ("uuid", "profile_uuid")
```

Comment:
I've usually used something like "get_profile_uuid" to make it clear that this is not an attribute, but a function. Maybe in this case it could be something like "get_profile_uuid_link"?

Reply:
Good point.

Reply:
Changed.